Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -164,5 +178,20 @@ public void updateMotorTelemetry() {
         leftDriveSlave.updateTelemetryProperties();
         rightDrive.updateTelemetryProperties();
         rightDriveSlave.updateTelemetryProperties();
+        
+        leftDistanceProp.set(leftDrive.getPosition());
+        rightDistanceProp.set(rightDrive.getPosition());
+    }
+    
+    public double getDistance() {
+        return convertToInches(leftDrive.getPosition() - startPosition.get());
+    }
+    
+    public void resetDistance() {
+        startPosition.set(convertToInches(leftDrive.getPosition()));
```

Comment:
The latter. I think that the property for telemetry purposes is good; it's just the use of such a property as a local immediate variable that makes me uncomfortable.